DRAFT.md

```diff
@@ -1651,3 +1651,26 @@ class Picture {
   void draw(Canvas& canvas) const;
 };
 ```
+
+## Appendix C: Enhancements vs breaking changes
+
+This table shows significant design decisions, the option taken and the cost and
+nature of changing the design. Design as presented in this paper has been
+approved by LEWG; we have until C++26 is standardized to consider making any
+breaking changes. After C++26, breaking changes are possible but with
+potentially significant user-impact.
+
+|Design decision|Current decision|Change|Nature of change| Breaking|
```

Comment:
Just a suggestion:

| Component / Factor / Choice / Item | Decision | Change | Change effect | Is breaking? |

Table caption or preceding text should make clear this is about design. 

In breaking column, replace 'Y' / 'N' with 'Yes' / 'No' or put 'Y'(es) / 'N'(o) (or similar) in breaking column heading so it is clear what 'Y' / 'N' mean.